README.rst

```diff
@@ -2,15 +2,30 @@ NetworkX
 ========
 
 
-.. image:: https://github.com/networkx/networkx/workflows/test/badge.svg?branch=main
-  :target: https://github.com/networkx/networkx/actions?query=workflow%3A%22test%22
+.. image::
+    https://github.com/networkx/networkx/workflows/test/badge.svg?
+    :target: https://github.com/networkx/networkx/actions
```

Comment:
Why were these links changed? For the badge, I think we want to make sure to show whether the main branch is passing or not. For the link, I think it's most useful that following the link shows actions that run the test suite since there are a lot of other actions.

Reply:
For the badge, I think I just went over it before committing and uniformized with the other badges ending in `?`. I agree with your suggestions otherwise; I'll change accordingly. Thanks!